.github/ISSUE_TEMPLATE/bug_report_android.yml

```
@@ -0,0 +1,96 @@
+name: 🐞 Bug Report - Android
+description: Tell us about something that's not working the way we (probably) intend.
+labels: ["Platform: Android", "Type: Bug"]
+body:
+  - type: dropdown
+    id: integration
+    attributes:
+      description: Integration
+      label: Integration
+      options:
+        - sentry-android
+        - sentry-android-ndk
+        - sentry-android-okhttp
+        - sentry-android-timber
+        - sentry-android-fragment
+        - sentry-apollo
+        - other
+    validations:
+      required: true
+
+  - type: dropdown
+    id: build_system
+    attributes:
+      description: Build system
+      label: Build System
+      options:
+        - Gradle
+        - Buck
+        - Bazel
+        - Other
+    validations:
+      required: true
+
+  - type: input
+    id: agp_version
+    attributes:
+      description: Android Gradle Plugin Version
+      placeholder: 7.0.0 ← should look like this
+      label: AGP Version
+    validations:
+      required: true
```

```suggestion
      required: false
```
Not mandatory right?